lib/JMAP/Tester.pm

```diff
@@ -195,8 +195,8 @@ has default_arguments => (
 
 =attr accounts
 
-This is an arrayref of accounts, provided by the client session object.  This
-method will return a list when accounts have been configured.
+This method will return a hash (not a hashref!) of accounts, as provided
```

Comment:
nothing ever returns a hash, only lists of pairs

Reply:
Updated